python/gigl/orchestration/kubeflow/kfp_orchestrator.py

```diff
@@ -176,6 +198,32 @@ def wait_for_completion(self, run: Union[aiplatform.PipelineJob, str]):
         Returns:
             None
         """
-        resource_name = run if isinstance(run, str) else run.resource_name
-        VertexAIService.wait_for_run_completion(resource_name)
-        logger.info(f"Pipeline run {resource_name} completed successfully.")
+        resource_names: list[str]
```

Comment:
```suggestion
        resource_names: list[str] = []
```

Why not just do this? If we really want the pre-declaration, why not this way? 

But again, I don't think this really makes the code more readable.

Reply:
done